### PR TITLE
docs/OnceFrom: Add some documentation around "once-from"

### PR DESCRIPTION
**Pull request by cgwalters**
This is an important bit that every MCO developer will need
to understand too.

Co-authored-by: Jonathan Lebon <jonathan@jlebon.com>
Co-authored-by: Kirsten G <kikis.github@gmail.com>

### CODE REVIEW

**Comment by jlebon** [comment]
This looks good to me, but I'll let someone who's not in the co-authored list do the needful. :)

**Comment by cgwalters** [comment]
@jlebon can you `/approve` here?

File: docs/OnceFrom.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# once-from
+
+The promise of the next-generation OpenShift installer and Red Hat CoreOS
+combined with the machine-config-operator is fully automated management
+of the hosts a highly integrated fashion.
```
Comment (kikisdeliveryservice) [inline docs/OnceFrom.md]: of the hosts in a....

File: docs/OnceFrom.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+However, we also aim to support a "bring your own host" model. We want to avoid
+as much as possible duplication between these two code paths. The idea of the
```
Comment (kikisdeliveryservice) [inline docs/OnceFrom.md]: duplication as much as possible..

File: docs/OnceFrom.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+"once-from" mode for the machine-config-daemon is that one can pass it an
+Ignition config (as generated by the installer for example) and execute it
+on on a host that was already created. (The feature also supports MachineConfig
```
Comment (kikisdeliveryservice) [inline docs/OnceFrom.md]: double "on"

**Comment by cgwalters** [comment]
> A few small changes. Thanks for writing this up!!

Fixed :arrow_up:

**Pull request merged**